guides/meetup.rst

```diff
@@ -176,7 +176,15 @@ Hodí se prozkoumat různá místa ve vašem městě a mít nějaké do zálohy,
 Informace na pyvo.cz
 ^^^^^^^^^^^^^^^^^^^^
 
-Před každým setkáním v předstihu přidejte informace na `pyvo.cz <https://pyvo.cz>`__. To lze udělat v **5 jednoduchých krocích**:
+Před každým setkáním v předstihu přidejte informace na `pyvo.cz <https://pyvo.cz>`__.
+Každý sraz je zaznamenán v `.yaml` souboru v `repozitáři na GitHubu <https://github.com/pyvec/pyvo-data>`__.
+
+Na stránce pro konkrétní město je odkaz *přidat informace o dalším srazu*,
+který otevře stránku s předvyplněnými informacemi pro další sraz.
+Doplň/odkomentuj co je potřeba a pošli jako Pull Request.
+
+
+Případně to lze udělat ručně v 5 krocích:
```

Comment:
Já bych tady nechal jen ten nejjednodušší způsob. Takže pokud to jde takhle, tak bych těch 5 kroků smazal. Co ty na to?